carlos comments for chapter 6 (#134)
chhoumann/p7: report/sections/Frontend/client-server-interaction.tex

In the interest of reducing the number of database queries, the result of this request is cached on the client.
The client will eagerly request new data to stay up to date with instructions or test code updates.
This short cache storage time is useful for when lecturers update test code during an exercise session.
This way, all students have the latest test code shortly after the update while still minimizing unnecessary queries.

> When a user submits an exercise solution, a HTTP POST request is sent to the Next.js backend.

Reviewer: I really disagree with changing mutation to HTTP POST request. We're using tRPC, so we use the terminology it uses.
Carlos wrote `what do you mean?` in his feedback, which tells me that he didn't read the preliminaries, where mutations are explained.

Author: maybe a compromise: a mutation is sent to the backend component in the form of an HTTP POST request.

Reviewer: No, that confuses the terminology

Reviewer: ```suggestion
When a user submits their exercise solution attempt, a mutation is sent to the Next.js backend.
```

In this request, the code and test is included.

> To handle this HTTP POST request, the Next.js server requests the Test Runner to process the given solution and test code.

Reviewer: I really disagree with changing mutation to HTTP POST request. We're using tRPC, so we use the terminology it uses.
Carlos wrote `what do you mean?` in his feedback, which tells me that he didn't read the preliminaries, where mutations are explained.

Reviewer: ```suggestion
To handle this mutation, the Next.js server requests the Test Runner to process the given solution and test code.
```
